torch/distributed/rpc/api.py

```diff
@@ -379,16 +379,18 @@ def _rref_typeof_on_user(rref):
 
 try:
     # Combine the implementation class and the type class.
-    class RRef(PyRRef, GenericWithOneTypeVar):
+    class RRef(PyRRef, Generic[T]):
```

Comment:
mypy complains " Variable "torch.distributed.rpc.api.GenericWithOneTypeVar" is not valid as a type  [valid-type]" if use GenericWithOneTypeVar here.